src/NovaTable.vue

```diff
@@ -1,6 +1,13 @@
 
 <template>
     <div class="nova-table clearfix">
+
+      <div style="position: absolute; left: 50%;">
+        <div style="position: relative; left: -50%;">
+            <slot name="top-center-bar"></slot>
+        </div>
+      </div>
+
```

Comment:
I checked this out to test it. Unfortunately, when you use it without any other top bar elements, the rest of the table bumps up and so there is overlap.

I tried it with this code:

```
    <nova-table
        :items="[{name: 'Fred', dob: '1980-01-02', role: 'Veterinarian'},
                 {name: 'Beth', dob: '1972-12-34', role: 'Customer'    },
                 {name: 'Gary', dob: '1998-09-08', role: 'Driver'      }]"
        :columns="{'name': 'Name', 'dob': 'Date of Birth', 'role': 'Job'}"
    >
        <div slot="top-center-bar">
            <h2>business time!</h2>
        </div>
    </nova-table>
```

Reply:
We'd want a solution that doesn't do that. Maybe the easiest choice is to just have the slot without any special alignment code and let the parent component have the alignment code.